.idea/modules.xml

```diff
@@ -2,8 +2,7 @@
 <project version="4">
   <component name="ProjectModuleManager">
     <modules>
-      <module fileurl="file://$PROJECT_DIR$/gwtp-idea-plugin.iml" filepath="$PROJECT_DIR$/gwtp-idea-plugin.iml" />
+      <module fileurl="file://$PROJECT_DIR$/gwtp-idea.iml" filepath="$PROJECT_DIR$/gwtp-idea.iml" />
     </modules>
   </component>
-</project>
-
+</project>
```

Comment:
I don't even know why those files are in the repo. I don't think the .idea folder should be there